aemedge/styles/styles.css

```diff
@@ -233,13 +241,12 @@ a.button::-moz-focus-inner {
 a.button.primary,
 button.primary,
 a.button.secondary,
-button.secondary {
+button.secondary,a.button.dark {
   border-radius: 1600px;
   opacity: 1;
-  min-width: 11.25rem;
   font-size: 1rem;
```

Comment:
Live site has font size of 18px for buttons; use this REM size (or if we have a variable for it
`font-size: 1.125rem;`